source/slang/slang-ir.h

```diff
@@ -1641,6 +1641,13 @@ struct IRRTTIPointerType : IRRawPointerTypeBase
     IR_LEAF_ISA(RTTIPointerType)
 };
 
+struct IRHLSLConstBufferPointerType : IRPtrTypeBase
+{
+    IR_LEAF_ISA(HLSLConstBufferPointerType)
+    IRInst* getElementType() { return getOperand(0); }
+    IRInst* getAlignment() { return getOperand(1); }
+};
```

Comment:
The alignment here is presumably the "element"/"value" alignment? Assuming that's right having it in the name might add to clarity.

Reply:
Okay I see ```All buffer reference addresses used for a particular buffer
      reference type are assumed to be aligned to this alignment value. That
      is, the base of the block must be aligned, and members of the block can
      be aligned within the block using standard layouts and offset layout
      qualifiers.```

So perhaps it should have name like Base alignment or something like that.

Reply:
Renaming done.